pydarn/plotting/fan.py

```diff
@@ -238,9 +214,115 @@ def plot_fan(cls, dmap_data: List[dict], ax=None, scan_index: int = 1,
                                          integer=True, nbins='auto')
             ticks = locator.tick_values(vmin=zmin, vmax=zmax)
 
-            cb = ax.figure.colorbar(mappable, ax=ax, extend='both', ticks=ticks)
+            cb = ax.figure.colorbar(mappable, ax=ax,
+                                    extend='both', ticks=ticks)
 
             if colorbar_label != '':
                 cb.set_label(colorbar_label)
 
-        return beam_corners_aacgm_lats, beam_corners_aacgm_lons, scan, grndsct, dtime
+        return beam_corners_aacgm_lats, beam_corners_aacgm_lons, scan, grndsct
+
+    @classmethod
+    def plot_fov(cls, stid: str, dtime: dt.datetime, ax=None,
+                 lowlat: int = 50, ranges: List = [0, 75],
```

Comment:
```suggestion
                 lowlat: int = 30, ranges: List = [0, 75],
```
following Devins suggestion